src/lang/english.txt

```diff
@@ -1012,6 +1012,7 @@ STR_GAME_OPTIONS_CURRENCY_IDR                                   :Indonesian Rupi
 STR_GAME_OPTIONS_CURRENCY_MYR                                   :Malaysian Ringgit
 STR_GAME_OPTIONS_CURRENCY_LVL                                   :Latvian Lats
 STR_GAME_OPTIONS_CURRENCY_PTE                                   :Portuguese Escudo
+STR_GAME_OPTIONS_CURRENCY_UAH                                   :Ukrainian Hryvnia
```

Comment:
This is correct, but you will also need to increment the `###length 44` on line 970 above. 🙂